packages/eslint-plugin/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.1.0 (Unreleased)
+
+### New Features
+
+- Add rule `@wordpress/no-unused-vars-before-return`
```

Comment:
This changelog should be in the other PR only right?

Reply:
> This changelog should be in the other PR only right?

Yep, sloppy rebase 😄

Reply:
> This changelog should be in the other PR only right?

Resolved in rebased e016b154e883671842e6302a74b451e06e84344d